docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+
+<p class="blog-description">64 out of 1589 answered</p>
+
+- **Customizability:** We already touched on this topic before and the frequency that it shows up on the MUI X space is a definitely a hint to double down on efforts to make customizations easier to do. We're aware that we might be missing some stuff, like documention around theming and a headless API.
+- **Cheaper Pro plan:** We know it can be out of range for a lot of developers and that's why the free version is already packed with many features. MUI mainly develops OSS software and we needed a way to sustain the project longer, hence the price we’re currently exercising, which actually is, still, way lower than similar products. However, many who left feedback around the price are individuals, so there might be an opportunity to evolve it for those not backed by their company.
+- **More features & fix features:** Collapsable rows, column resizing, and features for ERP apps were some of the features requested. And regarding features to fix, pagination with REST API, backend filtering, and cell editor, were the main mentioned.
+- **Improvements to the documentation:** Basically similar to the already discussed aspects of documentation improvement. Data Grid’s docs today could certainly use a better and optimized space and that’s we’ve been working on, so it’s easier to find and expand on the amount of information.
+- **Improvements to the look and feel:** It might be probably a solid idea to think of different default designs to offer on the Data Grid, alternative to Material Design. On a more UX side of things, we've heard many developers mentioning the filtering experience specifically.
+
+### How can we improve the Data Grid Pro for you?
+
+And now, here's what you mentioned about making the commercially licensed version of the Data Grid better:
+
+<img src="/static/blog/2021-survey/16.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="More features: 39.8%; Customizability: 20.4%; Fix features: 15.1%; Performance: 5.4%; Cheaper Premium plan: 3.2%; Improve docs: 3.2%; DX - API: 2.2%; LGPL license: 2.2%; More components: 2.2%" />
+
+<p class="blog-description">75 out of 1589 answered</p>
+
+- **More features:** [Grouping](https://mui.com/components/data-grid/group-pivot/#row-grouping) was the most requested feature for the Data Grid Pro, followed by [master detail](https://mui.com/components/data-grid/group-pivot/#master-detail), aggregation, search, [tree data](https://mui.com/components/data-grid/group-pivot/#tree-data), and [column pinning](https://mui.com/components/data-grid/columns/#column-pinning). Good thing is that we released some of these already during the time we were anlayzing the survey results. Check the links to see their docs and demos. Alternatively, you can follow [our public roadmap](https://github.com/mui/mui-x/projects/1) to see when the others will land.
```

Comment:
If "data grid" is lowercase but "Pro" plan gets a capital, then how do we feel about "Data Grid Pro"?

```suggestion
- **More features:** [Grouping](https://mui.com/components/data-grid/group-pivot/#row-grouping) was the most requested feature for the Data Grid Pro, followed by [master detail](https://mui.com/components/data-grid/group-pivot/#master-detail), aggregation, search, [tree data](https://mui.com/components/data-grid/group-pivot/#tree-data), and [column pinning](https://mui.com/components/data-grid/columns/#column-pinning). We've already released some of these in the time it took us to analyze the survey results. Check the links to see their docs and demos. Alternatively, you can follow [our public roadmap](https://github.com/mui/mui-x/projects/1) to see when the others will land.
```